src/Import/include/FMI1/fmi1_import_variable.h

```diff
@@ -41,44 +41,44 @@ extern "C" {
     fmi1_import_get_variable_by_name() and fmi1_import_get_variable_by_vr().
     @{
     */
-    /**@name Scalar variable types */
-/**@{ */
-/** \brief General variable type. 
-*
-* This type is convenient to unify all the variable list operations. 
-*    However, typed variables are needed to support specific attributes.
-*/
-typedef struct fmi1_xml_variable_t fmi1_import_variable_t;
-/** \brief  Opaque real variable */
-typedef struct fmi1_xml_real_variable_t fmi1_import_real_variable_t;
-/** \brief Opaque integer variable */
-typedef struct fmi1_xml_integer_variable_t fmi1_import_integer_variable_t;
-/** \brief Opaque string variable */
-typedef struct fmi1_xml_string_variable_t fmi1_import_string_variable_t;
-/** \brief Opaque enumeration variable */
-typedef struct fmi1_xml_enum_variable_t fmi1_import_enum_variable_t;
-/** \brief Opaque boolean variable */
-typedef struct fmi1_xml_bool_variable_t fmi1_import_bool_variable_t;
-/** \brief List of variables */
-typedef struct fmi1_import_variable_list_t fmi1_import_variable_list_t;
-/**@} */
-
-/**
-    \brief Get variable by variable name.
-    \param fmu - An fmu object as returned by fmi1_import_parse_xml().
-    \param name - variable name
-    \return variable pointer.
-*/
-FMILIB_EXPORT fmi1_import_variable_t* fmi1_import_get_variable_by_name(fmi1_import_t* fmu, const char* name);
-
-/**
-    \brief Get variable by value reference.
-    \param fmu - An fmu object as returned by fmi1_import_parse_xml().
-    \param baseType - basic data type
-    \param vr - value reference
-    \return variable pointer.
-*/
-FMILIB_EXPORT fmi1_import_variable_t* fmi1_import_get_variable_by_vr(fmi1_import_t* fmu, fmi1_base_type_enu_t baseType, fmi1_value_reference_t vr);
+    /**@name Scalar variable types */
+/**@{ */
```

Comment:
Is this only formatting changes?

Reply:
These show up very weird indeed, but aside from \param to @param, these are only whitespace changes.